### PR TITLE
Add Shiksha  my-courses 

### PR DESCRIPTION
**Pull request by harshad00**
`/shiksha/my-courses`
Screenshot 2024-10-28 092040

### CODE REVIEW

File: src/constant/global.ts
Comment (imsks) [inline src/constant/global.ts:1]: this change is not Needed for My Course. Always work on 1 task for 1 PR.

File: src/constant/pages/index.ts
Comment (imsks) [inline src/constant/pages/index.ts:1]: this change is not Needed for My Course. Always work on 1 task for 1 PR.

File: src/constant/routes.ts [hunk 1]
```diff
@@ -12,18 +12,23 @@ const routes = {
   },
   shiksha: '/shiksha',
   shikshaExplore: '/shiksha/explore',
+  shikshaMycu: '/shiksha/courses',
```
Comment (imsks) [inline src/constant/routes.ts:15]: this not needed. Explore page already exists.

File: src/constant/routes.ts [hunk 2]
```diff
@@ -12,18 +12,23 @@ const routes = {
...
+  shikshaCourses: '/shiksha/my-courses',
```
Comment (imsks) [inline src/constant/routes.ts:16]: name it like shikshaUserCourses

File: src/constant/routes.ts [hunk 3]
```diff
@@ -12,18 +12,23 @@ const routes = {
...
   interviewPrep: '/interview-prep',
   interviewPrepExplore: '/interview-prep/explore',
+
   workshops: '/workshops',
   os: '/os',
   contactUs: '/contact',
   internals: {
     landing: {
       products: 'products',
+      shiksha_courses: 'shiksha_courses',
```
Comment (imsks) [inline src/constant/routes.ts:31]: Not sure if needed. Even if needed then follow the same code style. write like shikshaCourses.

File: src/interfaces/global.ts [hunk 4]
```diff
@@ -29,6 +29,7 @@ export type ProductLabelType =
   | 'Roadmaps'
   | 'The Boring Projects'
   | 'Shiksha'
+  | 'Interview Prep'
```
Comment (imsks) [inline src/interfaces/global.ts:32]: this change is not Needed for My Course. Always work on 1 task for 1 PR.

File: src/pages/interview-prep/index.tsx [hunk 5]
```diff
@@ -40,3 +40,4 @@ const Home = ({ seoMeta }: PageProps) => {
 export const getServerSideProps = getPreFetchProps;
 
 export default Home;
+
```
Comment (imsks) [inline src/pages/interview-prep/index.tsx:43]: Remove empty lines.

File: src/pages/shiksha/my-courses/index.tsx
Comment (imsks) [inline src/pages/shiksha/my-courses/index.tsx:1]: This is wrong implementation. You'll need to call Get User Courses API and show Courses.

**Review by imsks — CHANGES_REQUESTED** [review]

**Comment by imsks** [comment]
@harshad00 Please work on every comment and fix them. Your implementation is wrong. check how we're fetching All Courses. All you've to do is just Implement it for One User.

**Comment by imsks** [comment]
@harshad00 Always resolve the comments like 
image

**Comment by harshad00** [comment]
I mistakenly made this branch from  `TWE_Web#59` that because it had unnecessary changes.  can I Close this PR and make new  branch from `production`  and male new PR.

**Comment by imsks** [comment]
Pick another task. This one is picked internally

**Pull request closed without merging**